DuckDuckGo/TabViewController.swift

```diff
@@ -2416,18 +2378,6 @@ extension TabViewController: SecureVaultManagerDelegate {
                 }
             }
 
-            if !autoSavedCredentialsId.isEmpty {
-                if let accountIdInt = Int64(autoSavedCredentialsId) {
-                    /// generatedPassword has been modified by user so delete the autosaved credential from db and prompt to save login as new credentials
-                    deleteLoginFor(accountIdInt: accountIdInt)
```

Comment:
Also this func `deleteLoginFor` can be deleted too since now since nothing is calling it